README.rst

```diff
@@ -406,6 +406,80 @@ information by performing lookups on the OUI
 
 This feature is enabled by default.
 
+Registering / Unregistering Chart Configuration
+-----------------------------------------------
+
+**OpenWISP Monitoring** provides registering and unregistering chart configuration through utility functions
+``openwisp_monitoring.monitoring.charts.register_chart`` and ``openwisp_monitoring.monitoring.charts.unregister_chart``.
+Using these functions you can register or unregister chart configuration from anywhere in your code.
+
+register_chart
+~~~~~~~~~~~~~~
+
+This function is used to register a new chart configuration from anywhere in your code.
+
++-------------------+-----------------------------------------------------+
+|     Parameter     |                     Description                     |
++-------------------+-----------------------------------------------------+
+|     chart_name    | A ``str`` defining name of the chart configuration. |
++-------------------+-----------------------------------------------------+
+|chart_configuration| A ``dict`` defining configuration of the chart.     |
++-------------------+-----------------------------------------------------+
+
+An example usage has been shown below.
+
+.. code-block:: python
+
+    from openwisp_monitoring.monitoring import register_chart
+
+    # Define configuration of your chart
+    chart_config = {
+        'type': 'histogram',
+        'title': 'Histogram',
+        'description': 'Histogram',
+        'top_fields': 2,
+        'order': 999,
+        'query': {
+            'influxdb': (
+                "SELECT {fields|SUM|/ 1} FROM {key} "
+                "WHERE time >= '{time}' AND content_type = "
+                "'{content_type}' AND object_id = '{object_id}'"
+            )
+        },
+
+    # Register your custom chart configuration
+    register_chart('chart_name', chart_config)
+
+.. note::
+
+    It will raise ``ImproperlyConfigured`` exception if a chart_configuration is already registered
+    with same name (not to be confused with verbose_name).
+
+unregister_chart
+~~~~~~~~~~~~~~~~
+
+This function is used to unregister a chart configuration from anywhere in your code.
+
++-------------+-----------------------------------------------------+
+|  Parameter  |                     Description                     |
++-------------+-----------------------------------------------------+
+|  chart_name | A ``str`` defining name of the chart configuration. |
++-------------+-----------------------------------------------------+
```

Comment:
I don't know what's wrong with the RST syntax here. I checked multiple places but still could not get it. Same thing is happening for `Signals` below.